### PR TITLE
docs: add buttons slot on PageHero component

### PR DESCRIPTION
**Pull request by CarolinaDelerue**

### CODE REVIEW

File: docs/content/design-kit.md [hunk 1]
```diff
@@ -18,6 +18,18 @@ Design Kit
...
+buttons:
+  - label: 'Download Design Kit'
+    size: 'lg'
+    variant: 'secondary'
+    to: '/assets/design-kit/nuxt-assets.zip'
+  - label: 'Preview guidelines'
+    size: 'lg'
+    to: 'https://www.figma.com/file/DOO3neicujKl3ymcqUg9dK/%E2%9B%B0%EF%B8%8F---Nuxt-Brand-Guidelines?node-id=0%3A1&t=qBS3UqvGpjrIkxIm-1'
+    icon: "uil:arrow-right"
+    trailing: 'trailing'
```
Comment (Flosciante) [inline docs/content/design-kit.md:31]: You can put this props directly on `::PageHero` after adding the buttons on `PageHero` component. (nuxtlabs/nuxt.com#1052)

**Review by Flosciante — CHANGES_REQUESTED** [review]

File: docs/content/3.showcase.md [hunk 2]
```diff
@@ -12,13 +12,32 @@ image:
   format: 'webp'
   height: '216'
   width: '400'
+textLeft: 'Download extension:'
+hidden: true
```
Comment (Flosciante) [inline docs/content/3.showcase.md:16]: Don't forget to update when you will have changed the names in the PR of nuxt.com (nuxtlabs/nuxt.com#1052)

File: docs/content/3.showcase.md [hunk 3]
```diff
@@ -12,13 +12,32 @@ image:
...
+buttons:
+  - label: 'Chrome'
+    size: 'lg'
+    variant: 'secondary'
+    to: "https://chrome.google.com/webstore/detail/vue-telescope/neaebjphlfplgdhedjdhcnpjkndddbpd"
```
Comment (Flosciante) [inline docs/content/3.showcase.md:21]: Simple quote

**Review by Flosciante — CHANGES_REQUESTED** [review]

**Review by Flosciante — APPROVED** [review]

**Review by clemcode — CHANGES_REQUESTED** [review]
TODO @clemcode : Revert yarn.lock

**Review by clemcode — APPROVED** [review]

**Pull request merged**